dashboard/fixtures/load_fixtures.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+clickhouse-client --host $CLICKHOUSE_HOST --database tensorzero --query "INSERT INTO JsonInference FORMAT CSV" < json_inference_examples.csv
+clickhouse-client --host $CLICKHOUSE_HOST --database tensorzero --query "INSERT INTO BooleanMetricFeedback FORMAT CSV" < boolean_metric_feedback_examples.csv
```

Comment:
At the end of this script, print that fixtures were applied successfully and the container will now exit with status code 0.